bucket/jj.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "0.12.0",
+    "description": "A Git-compatible DVCS that is both simple and powerful",
+    "homepage": "https://github.com/martinvonz/jj",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/martinvonz/jj/releases/download/v0.12.0/jj-v0.12.0-x86_64-pc-windows-msvc.zip",
+            "hash": "57e38681f6d8e4c6f5b6cf76a12325c85f4eb5a5a2b7e88ff8e279913c886404"
+        }
+    },
+    "bin": "jj.exe",
+    "checkver": {
+        "github": "https://github.com/martinvonz/jj"
+    },
```

```suggestion
    "checkver": "github",
```

Will be enough since github link already used as `homepage` (as per [documentation](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#special-cases)).